glib/src/subclass/types.rs

```diff
@@ -211,7 +212,7 @@ pub struct TypeData {
     #[doc(hidden)]
     pub parent_class: ffi::gpointer,
     #[doc(hidden)]
-    pub interface_data: *const Vec<(ffi::GType, ffi::gpointer)>,
+    pub class_data: *const HashMap<ffi::GType, ffi::gpointer>,
```

Comment:
```suggestion
    pub class_data: *const HashMap<ffi::GType, Box<dyn Any + Send + Sync>,
```

Reply:
Could also make it an `Option<ptr::NonNull<_>>` for cleanness

Reply:
> Could also make it an `Option<ptr::NonNull<_>>` for cleanness

It doesn't seem to like `ptr::NonNull<_>`, especially the `_` part : `the type placeholder `_` is not allowed within types on item signatures`

Reply:
Yes I was too lazy to type it out again. I meant `Option<ptr::NonNull<HashMap<ffi::GType, Box<dyn Any + Send + Sync>>>`